build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Would like to remove this file - will see if it was accidentally added by us and if so will delete.